SAMtemplates/apis/main.yaml

```diff
@@ -0,0 +1,304 @@
+                          #end
+                      #end
+                      #set($inputString = "$inputString }")
+                  #end
+
+                  ## Check if the request context should be included as part of the execution input
+                  #if($requestContext && !$requestContext.empty)
+                      #set($inputString = "$inputString,")
+                      #set($inputString = "$inputString @@requestContext@@: $requestContext")
+                  #end
+
+                  #set($inputString = "$inputString}")
+                  #set($inputString = $inputString.replaceAll("@@",'"'))
+                  #set($len = $inputString.length() - 1)
+                  "input": "{$util.escapeJavaScript($inputString.substring(1,$len))}"
+              }
+            - stateMachineArn: !Ref GetMyPrescriptionsStateMachineArn
+        IntegrationResponses:
+          - StatusCode: 200
+            ResponseTemplates:
+              application/json: |-
+                #set($payload = $util.parseJson($input.path('$.output')))
+                #set($context.responseOverride.status = $payload.Payload.statusCode)
+                #set($allHeaders = $payload.Payload.headers)
+
+                #foreach($headerName in $allHeaders.keySet())
+                    #set($context.responseOverride.header[$headerName] = $allHeaders.get($headerName))
+                #end
+
+                $payload.Payload.body
+      MethodResponses:
+        - StatusCode: "200"    
+
+  CapabilityStatementResource:
+    Type: AWS::ApiGateway::Resource
+    Properties:
+      RestApiId: !Ref RestApiGateway
+      ParentId: !GetAtt RestApiGateway.RootResourceId
+      PathPart: metadata
+
+  CapabilityStatementMethod:
+    Type: AWS::ApiGateway::Method
+    Properties:
+      RestApiId: !Ref RestApiGateway
+      ResourceId: !Ref CapabilityStatementResource
+      HttpMethod: GET
+      AuthorizationType: NONE
+      Integration:
+        Type: AWS_PROXY
+        Credentials: !GetAtt RestApiGatewayResources.Outputs.ApiGwRoleArn
+        IntegrationHttpMethod: POST
+        Uri: !Sub arn:aws:apigateway:${AWS::Region}:lambda:path/2015-03-31/functions/${CapabilityStatementFunctionArn}/invocations
+
+  StatusStatementResource:
+    Type: AWS::ApiGateway::Resource
+    Properties:
+      RestApiId: !Ref RestApiGateway
+      ParentId: !GetAtt RestApiGateway.RootResourceId
+      PathPart: _status
+  
+  StatusMethod:
+    Type: AWS::ApiGateway::Method
+    Properties:
+      RestApiId: !Ref RestApiGateway
+      ResourceId: !Ref StatusStatementResource
+      HttpMethod: GET
+      AuthorizationType: NONE
+      Integration:
+        Type: AWS_PROXY
+        Credentials: !GetAtt RestApiGatewayResources.Outputs.ApiGwRoleArn
+        IntegrationHttpMethod: POST
+        Uri: !Sub arn:aws:apigateway:${AWS::Region}:lambda:path/2015-03-31/functions/${StatusFunctionArn}/invocations
+
+  RestApiGatewayDeployment:
```

Comment:
can you add stage name to this and comment saying if you add a resource then change the stage name as we found we needed to do for _status endpoint in psu